content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/integrating_landed_costs.rst

```diff
@@ -1,81 +1,166 @@
+checkbox ticked, while the landed cost product's checkbox **is** ticked. This differentiates landed
+costs from all other costs displayed on the bill.
 
-.. image:: integrating_landed_costs/warehouse-transfer-landed-costs.png
+Additionally, at the top of the form, a :guilabel:`Create Landed Costs` button appears.
+
+.. image:: integrating_landed_costs/integrating-landed-costs-create-button.png
    :align: center
-   :alt: Use a warehouse transfer to cover a landed cost in the accounting journal.
+   :alt: Create Landed Costs button on vendor bill.
 
-After setting the picking, click :guilabel:`Compute` on the landed cost record. Then, go to the
-:guilabel:`Valuation Adjustments` tab to see the impact of the landed costs. Finally, click
-:guilabel:`Validate` to post the landed cost entry to the accounting journal.
+Add landed cost
+===============
 
-The user can access the journal entry that has been created by the landed cost by clicking on the
-:guilabel:`Journal Entry`.
+Once a landed cost is added to the vendor bill, click :guilabel:`Create Landed Costs` at the top of
+the vendor bill.
 
-.. note::
-   The product that the landed cost is applied to must have a product category set to a :abbr:`FIFO
-   (First In, First Out)` or an :abbr:`AVCO (Average Costing)` method.
+Doing so automatically creates a landed cost record with a set landed cost pre-filled in the product
+line in the :guilabel:`Additional Costs` tab.
 
-.. image:: integrating_landed_costs/landed-cost-journal-entry.png
+From here, click the :guilabel:`Transfers` drop-down menu, and select which transfer the landed cost
+belongs to.
+
+Once ready, click :guilabel:`Save` to save changes.
+
+.. image:: integrating_landed_costs/integrating-landed-costs-transfers-menu.png
    :align: center
-   :alt: Landed cost journal entry
+   :alt: Landed cost form with selected receipt transfer.
+
+.. tip::
+   In addition to creating landed costs directly from a vendor bill, landed cost records can *also*
+   be created by navigating to :menuselection:`Inventory app --> Operations --> Landed Costs`, and
+   clicking :guilabel:`New`.
+
+After setting the picking on the landed cost record, click :guilabel:`Compute` (at the bottom of the
+form, under the :guilabel:`Total:` cost).
+
+Click the :guilabel:`Valuation Adjustments` tab to see the impact of the landed costs. The
+:guilabel:`Original Value` column lists the original price of the |PO|, the :guilabel:`Additional
+Landed Cost` column displays the landed cost, and the :guilabel:`New Value` displays the sum of the
+two, for the total cost of the |PO|.
+
+Once ready, click :guilabel:`Validate` to post the landed cost entry to the accounting journal.
+
+This causes a :guilabel:`Valuation` smart button to appear at the top of the form. Click the
+:guilabel:`Valuation` smart button to open a :guilabel:`Stock Valuation` page, with the product's
+updated valuation listed.
```

Comment:
It might be good to add a note mentioning that you can view the valuation of every product, including landed costs, by navigating to `Inventory --> Reporting --> Valuation`.

Reply:
Suggestion implemented ✅